packages/cli-utils/package.json

```
@@ -49,6 +49,8 @@
   },
   "dependencies": {
     "@gql.tada/internal": "workspace:*",
+    "@0no-co/graphqlsp": "^1.8.0",
+    "ts-morph": "^22.0.0",
```

```suggestion
    "ts-morph": "~22.0.0",
```

Should we make this more strict?